client/curriculum/html/LinksAndButtons.tsx

```diff
@@ -0,0 +1,882 @@
+  { title: 'Link-uri vs butoane', id: 'linkuri-vs-butoane' },
+  { title: 'Exerciții', id: 'exercitii' },
+];
+
+export default function LinksAndButtonsLesson() {
+  return (
+    <>
+      <Head>
+        <title> Introducere | FrontEnd.ro</title>
+        <link rel="icon" href="/favicon.ico" />
+        <SEOTags
+          title="Link-uri și butoane | Lecție HTML"
+          description="Învață să adaugi link-uri și butoane în HTML și diferența dintre acestea."
+          url="https://FrontEnd.ro/html/linkuri-si-butoane"
+        />
+      </Head>
+      <Lesson
+        id="linkuriSiButoane"
+        title="Link-uri si butoane"
+        chapters={chapters}
+        withExercises
+      >
+        <LessonContributors className="absolute" contributors={contributors} />
+        <LessonCover>
+          {/* eslint-disable-next-line react/no-danger */}
+          <div dangerouslySetInnerHTML={{
+            __html: coverSvg,
+          }}
+          />
+        </LessonCover>
+        <p>
+          Deși la prima vedere pare un subiect simplu, sunt totuși multe de
+          știut despre link-uri și butoane în HTML. În acest articol vom discuta
+          despre markup, atribute și cele mai bune practici. Totodată vom trata
+          împreună cele două concepte pentru că de multe ori se folosesc
+          link-uri în loc de butoane și invers. Astfel la sfârșitul articolului
+          vom ști exact cum să implementăm un UX corect folosind cele 2
+          elemente.
+        </p>
+        <section>
+          <h2> The Basics </h2>
+          <LessonHeading as="h3" id="a-element">
+            {'<a> element'}
+          </LessonHeading>
+          <p>
+            Elementul
+            {' '}
+            <strong className="formatted">{'<a>'}</strong>
+            {' '}
+            denumit și
+            {' '}
+            <i>ancoră</i>
+            {' '}
+            sau
+            {' '}
+            <i>link</i>
+            {' '}
+            este unul din cele mai uzuale elemente
+            folosite pentru construirea paginilor web. Dacă vrem să ne mutam la
+            o altă pagină sau la o altă zonă în cadrul aceleiași pagini, atunci
+            vom folosi acest element. Modalitatea prin care adăugăm un link
+            într-o pagină este prin folosirea atributului
+            {' '}
+            <strong className="formatted">href</strong>
+            {' '}
+            care indică URL-ul
+            link-ului.
+          </p>
+        </section>
+        <section>
+          <LessonHeading as="h3" id="exemple-unde-folosim-tagul-a">
+            Exemple unde folosim tag-ul a
+          </LessonHeading>
+          <LessonHeading as="h4" id="cele-mai-uzuale">
+            Cele mai uzuale
+          </LessonHeading>
+          <Highlight
+            className="my-5"
+            language="html"
+            code={`
+<a href="https://frontend.ro/"> Frontend.ro </a>`}
+          />
+          <LessonFigure
```

Comment:
Move to an "inline" demo, meaning you just add the link in the article text and specify (maybe with bold text) "This is an absolute link". Remove the demo page for absolute links, I think it's pretty useless.